spec/asyncapi.md

```diff
@@ -529,167 +529,197 @@ defaultContentType: application/json
+<a name="channelObjectTags"></a>tags | [Tags Object](#tagsObject) | A list of tags for logical grouping of channels.
+<a name="channelObjectExternalDocs"></a>externalDocs | [External Documentation Object](#externalDocumentationObject) | Additional external documentation for this channel.
+
 
 This object can be extended with [Specification Extensions](#specificationExtensions).
 
-##### Channel Item Object Example
+##### Channel Object Example
 
 ```json
 {
-  "description": "This channel is used to exchange messages about users signing up",
-  "subscribe": {
-    "summary": "A user signed up.",
-    "message": {
-      "description": "A longer description of the message",
-      "payload": {
-        "type": "object",
-        "properties": {
-          "user": {
-            "$ref": "#/components/schemas/user"
-          },
-          "signup": {
-            "$ref": "#/components/schemas/signup"
+  "channels": {
+    "userEvents": {
+      "address": "users.{userId}",
+      "addressDelimiter": ".",
+      "description": "This channel is used to exchange messages about user events.",
+      "messages": {
+        "userSignedUp": {
+          "$ref": "#/components/messages/userSignedUp"
+        },
+        "userCompletedOrder": {
+          "$ref": "#/components/messages/userCompletedOrder"
+        }
+      },
+      "parameters": {
+        "userId": {
+          "$ref": "#/components/parameters/userId"
+        }
+      },
+      "servers": ["rabbitmqInProd", "rabbitmqInStaging"],
+      "bindings": {
+        "amqp": {
+          "is": "queue",
+          "queue": {
+            "exclusive": true
           }
         }
-      }
-    }
-  },
-  "bindings": {
-    "amqp": {
-      "is": "queue",
-      "queue": {
-        "exclusive": true
+      },
+      "tags": [{
+        "name": "user",
+        "description": "User-related messages"
+      }],
+      "externalDocs": {
+        "description": "Find more info here",
+        "url": "https://example.com"
       }
     }
   }
 }
 ```
 
 ```yaml
-description: This channel is used to exchange messages about users signing up
-subscribe:
-  summary: A user signed up.
-  message:
-    description: A longer description of the message
-    payload:
-      type: object
-      properties:
-        user:
-          $ref: "#/components/schemas/user"
-        signup:
-          $ref: "#/components/schemas/signup"
-bindings:
-  amqp:
-    is: queue
-    queue:
-      exclusive: true
+channels:
+  userEvents:
+    address: 'users.{userId}'
+    addressDelimiter: '.'
+    description: This channel is used to exchange messages about user events.
+    messages:
+      userSignedUp:
+        $ref: '#/components/messages/userSignedUp'
+      userCompletedOrder:
+        $ref: '#/components/messages/userCompletedOrder'
+    parameters:
+      userId:
+        $ref: '#/components/parameters/userId'
+    servers:
+      - rabbitmqInProd
+      - rabbitmqInStaging
```

Comment:
That change shouldn't be related to this PR, but shouldn't we also allow refs in channel's `servers`? I see that instead of IDs here references are used for channels and messages, so for servers they should too, like:

```yaml
servers:
  - $ref: '#/servers/rabbitmqInProd'
  - $ref: '#/servers/rabbitmqInStaging'
```

I think we even had a discussion on this topic once at some 3.0.0 meeting but I can't remember at which one.

WDYT? cc @smoya

Reply:
I think you mean my mention in https://github.com/asyncapi/spec/pull/806#discussion_r900012119 right?

Reply:
Exactly, the same logic, but I remember that there was a discussion about it at some meeting about the servers themselves.

Reply:
Yeah, if we end up accepting this PR, we'll have to open an issue to change everything else too. But it also really depends on the output of https://github.com/asyncapi/spec/issues/663.